tensorflow/lite/micro/kernels/xtensa_hifimini/fully_connected.cc

```diff
@@ -202,11 +229,32 @@ TfLiteStatus EvalQuantizedInt8(TfLiteContext* context, TfLiteNode* node,
   op_params.quantized_activation_min = data.output_activation_min;
   op_params.quantized_activation_max = data.output_activation_max;
 
-  xtensa::hifimini::FullyConnected(
-      op_params, GetTensorShape(input), GetTensorData<int8_t>(input),
-      GetTensorShape(filter), GetTensorData<int8_t>(filter),
-      GetTensorShape(bias), GetTensorData<int32_t>(bias),
-      GetTensorShape(output), GetTensorData<int8_t>(output));
+  {
+    int ret, b, weight_depth, out_depth, batches;
+    int8_t * p_out = GetTensorData<int8_t>(output);
+    weight_depth = GetTensorShape(filter).Dims(GetTensorShape(filter).DimensionsCount()-1);
+    out_depth = GetTensorShape(output).Dims(GetTensorShape(output).DimensionsCount()-1);
+    batches = FlatSizeSkipDim(GetTensorShape(output), GetTensorShape(output).DimensionsCount()-1);
+
+    //TODO: Use xa_nn_fully_connected_sym8xasym8s_asym8s? the kernel tests fail with it.
```

Comment:
This is probably a bug in the kernel tests.  I will have a a look internally at why we are testing against asymmetric weights for fully_connected_test.

Reply:
Please let me know if there is any update